pkg/operator/context/raw_features.go

```diff
@@ -30,13 +30,17 @@ import (
 func getRawFeatures(
 	config *userconfig.Config,
 	env *context.Environment,
+	pythonPackages context.PythonPackages,
 ) (context.RawFeatures, error) {
 
 	rawFeatures := context.RawFeatures{}
 
 	for _, featureConfig := range config.RawFeatures {
 		var buf bytes.Buffer
 		buf.WriteString(env.ID)
+		for _, pythonPackage := range pythonPackages {
+			buf.WriteString(pythonPackage.GetID())
```

Comment:
I think we should add the PythonPackage IDs directly to the ID calculation for aggregates, transformed_features, and models just to be safe, since those are the resources that directly use the PythonPackages

Reply:
adding dependency to aggregators, transformers instead of aggregates and transformed_features

Reply:
Good call!